### ISSUE TITLE
CMS users doesn't get notified why they can't remove their last method when MFA is enforced

### ISSUE THREAD

**Issue by newleeland** [silverstripe/silverstripe-mfa#207]
In the event that MFA is enforced and only one method is registered by the user, the user does not get any notification why they can't remove their last method in their profile, when pressing the "remove" link under their registered MFA method.

**Solution** 
- Remove the 'Remove method' button if MFA is enforced and there is only one registered method for the member.

~Consider using toast notification OR Banner Alert with text:
"Unable to remove this method, as MFA is required for login"~

**Comment by robbieaverill** [comment]
Can we just remove the UI option to remove it if it’s the last method?

**Comment by clarkepaul** [comment]
If we don't show any tip as to why it can't be removed we should at least make sure we cover this in the user help. 

If MFA is a requirement we could also have a permanent note in a users profile by the MFA fields "MFA is compulsory on this site, at least one authentication method will be required." as they might be aware its a requirement? Other words to use could be; required, enforced, mandatory... if they suit better.

**Comment by brynwhyman** [comment]
We'll raise another issue to give the user more information that MFA is enforced

**Comment by NightJar** [comment]
Adding a notice about why a user cannot remove the last method on their account (while MFA is required) is currently blocked by #249

**Comment by brynwhyman** [comment]
This issue is pointed as a 1, because we had previously made the decision to proceed with the stated AC:
'Remove the 'Remove method' button if MFA is enforced and there is only one registered method for the member.'

I'd expect this would mitigate #249 from even happening?

**Comment by NightJar** [comment]
Resolved in #252
